rocky/reports/report_types/aggregate_organisation_report/appendix.html

```diff
@@ -5,7 +5,15 @@
         <h2 id="appendix">{% translate "Appendix" %}</h2>
         {% include "aggregate_organisation_report/term_overview.html" %}
 
-        <h3 id="selected-oois">{% translate "Selected objects" %}</h3>
+        <h3 id="selected-oois">{% translate "Selected Objects" %}</h3>
```

Comment:
I don't think `objects` should be capitalized?

```suggestion
        <h3 id="selected-oois">{% translate "Selected objects" %}</h3>
```

Reply:
Don't know for sure, I see that we don't use camel casing voor headers, so I think this is ok @HeleenSG what do you think?